examples/dyldinfo.rs

```diff
@@ -126,8 +122,7 @@ fn main () {
                     "-bind" => { bind = true }
                     "-lazy_bind" => { lazy_bind = true }
                     other => {
-                        println!("unknown flag: {}", other);
-                        println!("");
+                        println!("unknown flag: {}\n", other);
```

Comment:
Which clippy warning is this?

Reply:
FYI, it is [print_with_newline](https://rust-lang.github.io/rust-clippy/master/index.html#print_with_newline).

Reply:
Ah okay, please use `println!()` instead. It's clearer and I don't think the performance is a concern.

Reply:
Also I think you meant [println_empty_string](https://rust-lang.github.io/rust-clippy/master/index.html#println_empty_string).

Reply:
If we revert this change, every time clippy run it will warn about this. 
Kind of annoyance.

Reply:
Ok, I forget I could just drop the empty string.